demos/spm_face_rep/face_rep_resolution.m

```diff
@@ -39,6 +42,16 @@
                                          'derivatives', ...
                                          ['cpp_spm-res' num2str(iResolution)]), 'cpath');
 
+  content = spm_jsonread(opt.model.file);
+  content.Name = [content.Name, ' resolution - ', num2str(iResolution)];
+
+  p = bids.internal.parse_filename(modelFile);
+  p.model = [p.model, ' resolution', num2str(iResolution)];
+  newModel = spm_file(opt.model.file, 'filename', createFilename(p));
+  opt.model.file = newModel;
+
+  spm_jsonwrite(newModel, content, struct('indent', '   '));
+
```

Comment:
this is only for this function to create names (in order to do so, we need to create different model.json files) for different resolution? so that spatial preprocssing folders would be different?

Reply:
Yeah this is only to create a different GLM folder name.

This is a bit "ugly" because everything in those json is the same: but I was trying to find a way to create different folder names that would not involve adding another option to the pile and would instead reuse something we already have to create GLM folder names.

Does that make sense? 

Another option could have been to have a different `cpp_spm-stats` folder for each resolution but then all the stats would not be centralized.

I think there is only so much data curation that can be automatized.